pkg/manager/daemon_adaptor.go

```diff
@@ -145,10 +148,20 @@ func (m *Manager) BuildDaemonCommand(d *daemon.Daemon, bin string, upgrade bool)
 				return nil, errors.Wrapf(err, "locate bootstrap %s", bootstrap)
 			}
 
-			cmdOpts = append(cmdOpts,
-				command.WithConfig(d.ConfigFile("")),
-				command.WithBootstrap(bootstrap),
-			)
+			if config.IsConfigSourceEnabled() {
+				configApiPath := fmt.Sprintf(endpointGetDaemonConfig, d.States.ID)
+				cmdOpts = append(cmdOpts,
+					command.WithConfig(configApiPath),
```

Comment:
Is the `configApiPath` a local config file?

Reply:
configApiPath is a URI path